Makefile

```diff
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
 # version is moved to a separate repo and release process.
-export IMAGE_VERSION = v1.4.0
+export IMAGE_VERSION = v1.5.0
 # Build-time variables to inject into binaries
```

Comment:
/hold
Can we get merged a few docs before the release? 
e.g https://github.com/operator-framework/operator-sdk/pull/4640